pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -127,6 +127,9 @@ type LocalSecretReference struct {
 const (
 	// ClusterConditionReady means the cluster is healthy and ready to accept workloads.
 	ClusterConditionReady = "Ready"
+	// ClusterMetadataSufficient means some err occurs when access member cluster but it doesn't affect the use of member cluster
+	// ClusterMetadataSufficient are ready to accept workloads too
+	ClusterMetadataSufficient = "MetadataSufficient"
```

Comment:
```suggestion
	// ClusterMetadataSufficient means some metadata of the cluster is unavailable, such as Kubernetes version, or
	// API enablement list.
	// Clusters in insufficient mode condition(MetadataSufficient=false) are still ready to accept workloads,
	// but the scheduler might be favor to cluster in sufficient condition(MetadataSufficient=true), that depends on
	// scheduler implementation.
	ClusterMetadataSufficient = "MetadataSufficient"
```
How about this? 
(And, don't get me wrong, I don't mean this PR should take care of the scheduler, this PR just focus on how to maintain the condition.)

Reply:
fixed